source/getting-started/js-primer.md

```diff
@@ -52,3 +52,50 @@ export default Ember.Component.extend({
   }
 });
 ```
+
+## Object Literal Shorthand
+
+In ES6 some functionality was added that allows the use of terser syntax for object literal properties and methods, along with some enhanced functionality for property names.
```

Comment:
What do you think of "ES6 introduced a new syntax for object literals ..." as the opening line. I'm not too comfortable with `some functionality was added`. If that's just me feel free to ignore this comment.